### DISCUSSION TITLE
Update button

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#2243] [Ideas]
Can you make it so the update button turns red when there are updates. That would be nice to have.

**Comment by seerge** [comment]
@Reddot911 hello, it would force app to download drivers list and also run a full version comparison with device manager.

It’s quite heavy operation, and i don’t think it makes sense to run it every time.

Driver updates are quite rare anyway, and GPU drivers you get directly from AMD / NVidia

**Reply by ghost** [reply]
for sure. keep up the great work. Curious is there any future updates for the rog duo 16? For instance increase wattage for gpu to increase performace or anything else?

**Reply by seerge** [reply]
@Reddot911 features depend if your device supports them in a first place or not :) GPU power limits is something Asus has added only in 2024

**Reply by ghost** [reply]
Okay, reason I ask about wattage just been reading about people vbios flash their gpu from a similar specs to increase wattage say 110 to 130.
